src/main/java/com/stripe/model/CountrySpec.java

```diff
@@ -27,29 +27,45 @@
   List<String> supportedPaymentMethods;
   VerificationFields verificationFields;
 
+  // <editor-fold desc="list">
+  /**
+   * List Country Specs.
```

Comment:
Is there a particular rationale for deciding when to capitalize resources versus not? I noticed that "charge" above is not capitalized while "Apple Pay" is, which is correct, but I would've though that "Country Spec", which isn't a proper noun, would be "country spec" if it was following the same rules. I guess it's a weird one though.

Reply:
I just copied/pasted from the API reference, but I agree that "country specs" probably shouldn't be capitalized.